### ISSUE TITLE
artifact_deploy procs don't change cwd

### ISSUE THREAD

**Issue by promisedlandt** [RiotGamesCookbooks/artifact-cookbook#25]
From the readme

``` ruby
 migrate Proc.new {
    execute "bundle exec rake db:migrate" do
      environment { 'RAILS_ENV' => 'production' }
      user "riot"
      group "riot"
    end
  }
```

does not work for me, because it is executed in  `/srv/pvpnet`, and not in `/srv/pvpnet/releases/whatever`.

Since the procs are just `recipe_eval`'ed, it seems hard to change the execute block in the proc to add the cwd automatically, but then the example in the readme should be changed to 

``` ruby
 migrate Proc.new {
    execute "bundle exec rake db:migrate" do
      cwd release_path
      environment { 'RAILS_ENV' => 'production' }
      user "riot"
      group "riot"
    end
  }
```

Same thing for before_migrate proc.

**Comment by reset** [comment]
thanks @promisedlandt we'll take a look at this one

**Comment by KAllan357** [comment]
This should be addressed in #40. For now, I've updated the documentation to be more clear. We can definitely revisit this issue if we decide its more important in the future.
